shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -0,0 +1,342 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
(I skimmed this, I'm assuming the core logic isn't changed from the FlutterMain blocks.)

Reply:
As far as I know, you are correct in that assumption.